src/commands/cmd_cluster.cc

```diff
@@ -253,6 +278,10 @@ class CommandClusterX : public Commander {
   int64_t slot_ = -1;
   int slot_id_ = -1;
   bool force_ = false;
+
+  bool sync_migrate_ = false;
+  int sync_migrate_timeout_ = 0;
+  std::shared_ptr<SyncMigrateContext> sync_migrate_ctx_ = nullptr;
```

Comment:
@ellutionist Great job!
I just thinking if we need to have `sync_migrate_ctx_` inside CommandClusterX because you just create it and pass it to the `MigrateSlot` function without further use of this object.